src/service/Data.tsx

```diff
@@ -118,8 +118,8 @@ const processTimestamps = (records: Modite[] = [], date: Date) => {
   const nowUtc: Date = new Date(date.getTime() + date.getTimezoneOffset())
 
   records.forEach((item: Modite) => {
-    const itemDate: Date = new Date((nowUtc as any) - (item.tz_offset as number) * 60000)
-
+    var CAoffset = -3
+    const itemDate: Date = new Date(nowUtc as any)
```

Comment:
With typescript, `any` is almost always considered bad. You have explicitly set it here which is better than implicit but you should be more specific of what `nowUtc` is. Looks like it is defined on line 118 and it's a `Date` so should replace `any` with `Date`.